style.css

```diff
@@ -6,468 +6,975 @@
+    background-image: url("./images/Ellipse\ 20@2x.png");
+    background-position: relative;
+    background-size: 15%;
+    background-repeat: no-repeat;
+    background-position-x: right;
+    background-position-y: center;
+  }
 
-.detail-input {
-  display: flex;
-  align-items: center;
-  padding: 10px;
-  font-family: 'Inter', sans-serif;
-  font-weight: 400;
-  font-size: 15px;
-  line-height: 24px;
-  background-color: #f7f3ff;
-  width: 100%;
-  margin: 10px 0;
-  border-style: none;
-}
+  .nav-bar {
+    display: flex;
+    max-width: 100%;
+    height: 75px;
+    justify-content: space-between;
+    align-items: center;
+    background-color: #00005b;
+  }
 
-.submit-btn {
-  cursor: pointer;
-  color: #edecff;
-  font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
-  font-style: normal;
-  padding: 12px 19px;
-  font-weight: 500;
-  background-color: #7d73fc;
-  border-style: none;
-  font-size: 12px;
-  letter-spacing: 0.04em;
-  box-shadow: rgba(125, 115, 252, 0.48);
-}
+  .nav-item > a {
+    gap: 10px;
+    margin-right: 20px;
+    justify-items: center;
+    align-items: center;
+    text-decoration: none;
+    font-size: 22px;
+    font-weight: 400;
+    color: rgb(207, 184, 184);
+    font-family: 'Roboto', sans-serif;
+  }
 
-.submit-btn:active {
-  background-color: #5e42d7;
-}
+  .mail-icon {
+    margin-bottom: 0;
+  }
 
-a:hover,
-button:hover {
-  color: burlywood;
-}
+  .nav-logo {
+    padding-left: 20px;
+    text-decoration: none;
+    color: rgb(207, 184, 184);
+    font-size: 24px;
+    font-weight: 500;
+    margin-bottom: 10px;
+  }
 
-.btn-container {
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  text-align: center;
-  margin: 20px 0;
-}
+  .page-content {
+    display: flex;
+    flex-direction: column;
+    max-width: 100%;
+    min-height: 100%;
+    justify-items: center;
+    align-items: center;
+    justify-content: center;
+    background-image: url('./images/circle\ 1\ header@2x.jpg');
+    background-position: relative;
+    background-size: 22%;
+    background-repeat: no-repeat;
+    background-position-x: left;
+    background-position-y: top;
+  }
 
-.end-line {
-  display: block;
-  align-self: center;
-  border-bottom: 2px solid rgb(70, 67, 67);
-  margin: 10px 0 0 0;
-  width: 100%;
-}
+  .descript {
+    max-width: 507px;
+    display: flex;
+    flex-grow: 1;
+  }
 
-.email-me {
-  color: #ebebff;
-  text-decoration: none;
-  font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
-  font-style: normal;
-  font-weight: 400;
-  font-size: 20px;
-  line-height: 28px;
-  margin: 10px 0 20px 0;
-}
+  h1 {
+    font-size: 85px;
+  }
+
+  .name {
+    font-weight: 500;
+    color: #6495ed;
+    text-align: center;
+    margin: 40px 0;
+  }
+
+  .occupation {
+    color: #afabab;
```

Comment:
- [ ] Please change this color from `#afabab` to `#DFE1E6` so that it is same as the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A976).